packages/front-end/components/OpenVisualEditorLink.tsx

```diff
@@ -27,21 +27,33 @@ const OpenVisualEditorLink: FC<{
   const url = useMemo(() => {
     if (!visualEditorUrl) return "";
 
-    return appendQueryParamsToURL(visualEditorUrl, {
+    let url = visualEditorUrl;
+
+    // Force all URLs to be absolute
+    if (!url.match(/^http(s)?:/)) {
+      // We could use https here, but then it would break for people testing on localhost
+      // Most sites redirect http to https, so this should work almost everywhere
+      url = "http://" + url;
+    }
+
+    url = appendQueryParamsToURL(url, {
       "vc-id": id,
       "v-idx": changeIndex,
       "exp-url": encodeURIComponent(window.location.href),
       "api-host": encodeURIComponent(apiHost),
     });
+
+    return url;
   }, [visualEditorUrl, id, changeIndex, apiHost]);
 
   return (
     <>
-      <span
+      <a
+        href={url}
         className="btn btn-sm btn-primary"
         onClick={async (e) => {
+          e.preventDefault();
```

Comment:
Will changing to an anchor tag and removing `preventDefault` circumvent the checks we have implemented inside the `onClick` handler?

Reply:
It the user right clicks and opens in a new tab, it will bypass the extension check and tracking calls. I was mostly thinking about the experience if you are using a dev build of the chrome extension. The extension check will fail, and it would be nice if there was an easy way to bypass it without modifying code. Open to other ideas. Maybe in the extension modal we can add a small link to continue anyway?

I have a feeling I do have a bug there right now when URL is empty. Need to test this some more.